.github/workflows/build.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - name: Update binaries
         run: |
           $ErrorActionPreference = 'SilentlyContinue'
-          git clone --recursive https://github.com/uNetworking/uWebSockets.js.git
+          git clone --recursive https://github.com/${{ github.repository }}.git
```

Comment:
This was easier for when testing on my fork, but let me know if I should revert this and the others

Reply:
If it works this is better

Reply:
Should be just fine 👍